features/index/page_title.feature

```diff
@@ -34,7 +34,8 @@ Feature: Index - Page Title
     """
       ActiveAdmin.register Post do
         controller do
-          before_filter { @page_title = "List of #{resource_class.model_name.plural}" }
+          callback = ActiveAdmin::Dependency.rails >= 4 ? :before_action : :before_filter
+          send(callback) { @page_title = "List of #{resource_class.model_name.plural}" }
```

Comment:
maybe we should build a `alias_method :before_action, :before_filter if ActiveAdmin::Dependency.rails >= 4` into the `ActiveAdmin::BaseController`